### PR TITLE
Updated some English audios to Hindi  

### PR DESCRIPTION
**Pull request by ashvin89**
|PR checklist|Status|
|-----|:------:|
|Is issue created in GitHub?|[Yes](#128  )|Yes|
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|NA|
|Is code analysis task executed and issues resolved in the CPP/native code?|No|
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|NA|
|Is code commenting done properly for the changes made?|NA|
|Is correct variable naming convention followed?|NA|
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|NA|
|Are the message strings used in the application taken from strings.xml file?|NA|
|Are the permissions which are not required removed from AndroidManifest.java file?|NA|
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|NA|
|Are the generated files correctly configured in .gitignore?|NA|
|Are the release signing configurations removed from the gradle files?|NA|
|Is commented code removed from all files?|NA|
|Are updated images compressed (< 100 KB)?|NA|
|Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|NA|

### CODE REVIEW

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_13_ATriangleHas.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+A Triangle has three straight sides
```
Comment (Shailin) [inline English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_13_ATriangleHas.txt]: A **triangle** has three straight sides

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_20_ARectangleLooks.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+A Rectangle looks like an streched out square. All four corners are the same but not all the sides are of the same length. Opposite sides are of equal length
```
Comment (Shailin) [inline English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_20_ARectangleLooks.txt]: A **rectangle** looks like a stretched out square.

File: Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_22_DrawARectangle.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+एक rectangle ड्रा करें
```
Comment (Shailin) [inline Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_22_DrawARectangle.txt]: एक rectangle **draw** करें

File: Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_7_ItIsVery.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+एक पूर्ण circle बनाना बहुत कठिन है। अच्छा circle ड्रॉ करने की कोशिश करें.
```
Comment (Shailin) [inline Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_7_ItIsVery.txt]: अच्छा circle draw करने की कोशिश करें.
Also check the sentence.

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_10_ItsYourTurn.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+It's your turn
```
Comment (Shailin) [inline English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_10_ItsYourTurn.txt]: Add punctuations wherever applicable

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_20_ARectangleLooks.txt (hunk 2 above)
Comment (ashvin89) [reply English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_20_ARectangleLooks.txt]: Corrected

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_10_ItsYourTurn.txt (hunk 5 above)
Comment (ashvin89) [reply English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_10_ItsYourTurn.txt]: Corrected

File: Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_7_ItIsVery.txt (hunk 4 above)
Comment (ashvin89) [reply Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_7_ItIsVery.txt]: Checked the sentence, it is correct

File: Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_22_DrawARectangle.txt (hunk 3 above)
Comment (ashvin89) [reply Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_22_DrawARectangle.txt]: Corrected

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_13_ATriangleHas.txt (hunk 1 above)
Comment (ashvin89) [reply English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_13_ATriangleHas.txt]: Corrected

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_10_ItsYourTurn.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+It's you turn.
```
Comment (Shailin) [inline English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_10_ItsYourTurn.txt]: It's your turn.

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_21_ItsYourTurn.txt [hunk 7]
```diff
@@ -0,0 +1 @@
+It's you turn.
```
Comment (Shailin) [inline English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_21_ItsYourTurn.txt]: It's your turn.

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_2_TapTheStick.txt [hunk 8]
```diff
@@ -0,0 +1 @@
+Tap the stick to draw. Tap the brom to erase your marks.
```
Comment (Shailin) [inline English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_2_TapTheStick.txt]: Tap the stick to draw. Tap the **broom** to erase your marks.

File: Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_2_TapTheStick.txt [hunk 9]
```diff
@@ -0,0 +1 @@
+चित्र बनाने के लिए छड़ी टैप करें. अपने निशान मिटाने के लिए jhaadoo पर टैप करें.
```
Comment (Shailin) [inline Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_2_TapTheStick.txt]: चित्र बनाने के लिए छड़ी **पर** टैप करें

File: Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_5_ACircleIsA.txt [hunk 10]
```diff
@@ -0,0 +1 @@
+Circle  एक आकृति है, जो एक center बिंदु के चारों ओर एक curve खींचकर बनाया गया है. Circle का प्रत्येक बिंदु center बिंदु से समान दूरी पर है.
```
Comment (Shailin) [inline Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_5_ACircleIsA.txt]: Remove unwanted spaces

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_10_ItsYourTurn.txt (hunk 6 above)
Comment (ashvin89) [reply English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_10_ItsYourTurn.txt]: Corrected

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_2_TapTheStick.txt (hunk 8 above)
Comment (ashvin89) [reply English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_2_TapTheStick.txt]: Corrected

File: Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_5_ACircleIsA.txt (hunk 10 above)
Comment (ashvin89) [reply Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_5_ACircleIsA.txt]: Removed

File: Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_2_TapTheStick.txt (hunk 9 above)
Comment (ashvin89) [reply Hindi_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_2_TapTheStick.txt]: Corrected

File: English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_21_ItsYourTurn.txt (hunk 7 above)
Comment (ashvin89) [reply English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_21_ItsYourTurn.txt]: Corrected

**Review by Shailin — APPROVED** [review]

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**